Marlin/Marlin_main.cpp

```diff
@@ -1409,6 +1409,9 @@ bool get_target_extruder_from_command(const uint16_t code) {
           soft_endstop_max[axis] = base_max_pos(axis) + offs;
         }
       }
+    #elif ENABLED(DELTA)
+      soft_endstop_min[axis] = base_min_pos(axis) + (axis == Z_AXIS) ? 0 : offs;
```

Comment:
Note that because of order of operations, this will be evaluated as…
```cpp
(base_min_pos(axis) + (axis == Z_AXIS)) ? 0 : offs;
```
…so should be…
```cpp
base_min_pos(axis) + (axis == Z_AXIS ? 0 : offs);
```